src/rules/color-named/index.js

```diff
@@ -55,6 +56,11 @@ export default function (expectation, options) {
       valueParser(decl.value).walk(node => {
         const { value, type, sourceIndex } = node
 
+        if (
+          optionsMatches(options, "ignore", "inside-function")
+          && type === "function"
+          && !colorFunctionNames.has(value.toLowerCase())) { return false }
```

Comment:
@jeddy3 Do you think this line is important? Right now I'm thinking it should be cut. I think if it ever actually came up it would be more confusing than helpful to the end user. 

Besides this particular point I think the PR is ready to merge.